AdminUi/src/AdminUi.Infrastructure/Persistence/IServiceCollectionExtension.cs

```diff
@@ -48,6 +49,7 @@ public static IServiceCollection AddDatabase(this IServiceCollection services, A
                             sqlOptions.CommandTimeout(20);
                             sqlOptions.MigrationsAssembly(POSTGRES_MIGRATIONS_ASSEMBLY);
                             sqlOptions.EnableRetryOnFailure(options.RetryOptions.MaxRetryCount, TimeSpan.FromSeconds(options.RetryOptions.MaxRetryDelayInSeconds), null);
+                            sqlOptions.MigrationsHistoryTable(HistoryRepository.DefaultTableName, "AdminUi");
```

Comment:
Can you please add a Todo comment here (and at every other place at which we set the migrations table) that remembers us to remove this as soon as the bug is fixed? (add a link to the bug)